### PR TITLE
Enhance descriptions of the Split Region section

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Enhance descriptions of the Split Region section according to the feedbacks from the user support issues

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/6318
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@Joyinqin This docs-cn PR https://github.com/pingcap/docs-cn/pull/6318 does not have the `v5.1` label. Please remove it.

File: sql-statements/sql-statement-split-region.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ To solve the hotspot problem in the above scenario, TiDB introduces the pre-spli
 
 There are two types of Split Region syntax:
 
+- The syntax of even split:
+
 {{< copyable "sql" >}}
 
 ```sql
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:53]: Please add 4 indentations.

File: sql-statements/sql-statement-split-region.md [hunk 2]
```diff
@@ -54,6 +56,8 @@ SPLIT TABLE table_name [INDEX index_name] BETWEEN (lower_value) AND (upper_value
 
 `BETWEEN lower_value AND upper_value REGIONS region_num` defines the upper boundary, the lower boundary, and the Region amount. Then the current region will be evenly spilt into the number of regions (as specified in `region_num`) between the upper and lower boundaries.
 
+- The syntax of uneven split:
+
 {{< copyable "sql" >}}
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:61]: 4 indentations are needed.

**Comment by Joyinqin** [comment]
@TomShawn Updated. PTAL, thx~

File: sql-statements/sql-statement-split-region.md [hunk 3]
```diff
@@ -290,14 +294,20 @@ Splitting Regions for partitioned tables is the same as splitting Regions for or
     +-----------+-----------+---------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
     ```
 
-2. Use the `SPLIT` syntax to split a Region for each partition. In the following example, four Regions are split in the range of `[0,10000]`.
+2. Use the `SPLIT` syntax to split a Region for each partition. If you want to divide the data in the `[0,10000]` range of each partition into four Regions, the example statement is as follows:
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:297]:
```suggestion
2. Use the `SPLIT` syntax to split a Region for each partition. Suppose that you want to split the data in the `[0,10000]` range of each partition into four Regions. The example statement is as follows:
```

File: sql-statements/sql-statement-split-region.md [hunk 4]
```diff
@@ -323,17 +333,17 @@ Splitting Regions for partitioned tables is the same as splitting Regions for or
...
 
     ```sql
     split partition table t index idx between (1000) and (10000) regions 2;
     ```
 
-#### Split Region for a single partition
+#### Examples of Split Region for a single partition
 
-You can specify the partition to be split. See the following usage example:
+You can specify the partition to be split. If you want to create a Range table divided into three partitions, the example statement is as follows:
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:346]: This sentence should be in **Line 348**: Suppose that you want to create a Range partitioned table split into three partitions. The example statement is as follows:

File: sql-statements/sql-statement-split-region.md [hunk 5]
```diff
@@ -346,23 +356,23 @@ You can specify the partition to be split. See the following usage example:
         partition p3 values less than (MAXVALUE) );
     ```
 
-2. Split two Regions in the `[0,10000]` range of the `p1` partition:
+2. If you want to pre-split the data in the range [0,10000] of the `p1` partition into two Regions, the example statement is as follows:
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:359]:
```suggestion
2. Suppose that you want to split the data in the `[0,10000]` range of the `p1` partition into two Regions. The example statement is as follows:
```

File: sql-statements/sql-statement-split-region.md [hunk 6]
```diff
@@ -346,23 +356,23 @@ You can specify the partition to be split. See the following usage example:
...
 
     {{< copyable "sql" >}}
 
     ```sql
     split partition table t partition (p1) between (0) and (10000) regions 2;
     ```
 
-3. Split two Regions in the `[10000,20000]` range of the `p2` partition:
+3. If you want to pre-split the data in the range [10000,20000] of the `p2` partition into two Regions, the example statement is as follows:
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:367]:
```suggestion
3. Suppose that you want to split the data in the `[10000,20000]` range of the `p2` partition into two Regions. The example statement is as follows:
```

File: sql-statements/sql-statement-split-region.md [hunk 7]
```diff
@@ -382,7 +392,7 @@ You can specify the partition to be split. See the following usage example:
     +-----------+----------------+----------------+-----------+-----------------+------------------+------------+---------------+------------+----------------------+------------------+
     ```
 
-5. Split two Regions in the `[0,20000]` range of the `idx` index of `p1` and `p2` partitions:
+5. If you want to pre-split the [0,20000] range of the index `idx` of the `p1` and `p2` partitions into two Regions, the example statement is as follows:
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:395]:
```suggestion
5. Suppose that you want to split the `[0,20000]` range of the `idx` index of the `p1` and `p2` partitions into two Regions. The example statement is as follows:
```

File: sql-statements/sql-statement-split-region.md [hunk 8]
```diff
@@ -290,14 +294,20 @@ Splitting Regions for partitioned tables is the same as splitting Regions for or
...
-2. Use the `SPLIT` syntax to split a Region for each partition. In the following example, four Regions are split in the range of `[0,10000]`.
+2. Use the `SPLIT` syntax to split a Region for each partition. If you want to divide the data in the `[0,10000]` range of each partition into four Regions, the example statement is as follows:
 
     {{< copyable "sql" >}}
 
     ```sql
     split partition table t between (0) and (10000) regions 4;
     ```
 
+    In the above statement, `0` and `10000` respectively represent the `row_id` of the upper and lower boundaries corresponding to the hotspot data you want to break up.
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:305]:
```suggestion
    In the above statement, `0` and `10000` respectively represent the `row_id` of the upper and lower boundaries corresponding to the hotspot data you want to scatter.
```

File: sql-statements/sql-statement-split-region.md [hunk 9]
```diff
@@ -290,14 +294,20 @@ Splitting Regions for partitioned tables is the same as splitting Regions for or
...
+
+    > **Note:**
+    >
+    > This example only applies to scenarios where data hotspots are evenly distributed. If the hotspot data is unevenly distributed in the data range you specify, refer to the syntax of uneven split in [Split Regions for partitioned tables](#split-regions-for-partitioned-tables).
```
Comment (TomShawn) [inline sql-statements/sql-statement-split-region.md:309]:
```suggestion
    > This example only applies to scenarios where hotspot data is evenly distributed. If the hotspot data is unevenly distributed in a specified data range, refer to the syntax of uneven split in [Split Regions for partitioned tables](#split-regions-for-partitioned-tables).
```

File: sql-statements/sql-statement-split-region.md [hunk 10]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/dev/sql-statements/sql-statement-split-region/','/docs/dev/refe
 
 # Split Region
 
-For each new table created in TiDB, one Region is segmented by default to store the data of this table. This default behavior is controlled by `split-table` in the configuration file. When the data in this Region exceeds the default Region size limit, the Region starts to split into two.
+For each new table created in TiDB, one [Region](/tidb-storage.md#region) is segmented by default to store the data of this table. This default behavior is controlled by `split-table` in the configuration file. When the data in this Region exceeds the default Region size limit, the Region starts to split into two.
```
Comment (crazycs520) [inline sql-statements/sql-statement-split-region.md:9]:
```suggestion
For each new table created in TiDB, one [Region](/tidb-storage.md#region) is segmented by default to store the data of this table. This default behavior is controlled by `split-table` in the TiDB configuration file. When the data in this Region exceeds the default Region size limit, the Region starts to split into two.
```

**Review by crazycs520 — COMMENTED** [review]
REST LGTM

**Review by crazycs520 — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**